policyengine_it/variables/household/person/is_dependent_between_21_and_24.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_it.model_api import *
+
+
+class is_dependent_between_21_and_24(Variable):
+    value_type = bool
+    entity = Person
+    label = "Is between 21 and 24 years old"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        return (
+            person("age", period) >= 21
+            and person("age", period) < 24
+            and person("is_dependent", period) == True
```

Comment:
```suggestion
           age = person("age", period)
            meets_age_reguirement = p.younger =< age < p.older
            dependent = person("is_dependent", period)
           return meets_age_reguirement & dependent
```
We try to define each "action" in separate variables e.g. above

Reply:
p stands for the parameter path: 
What we commonly use is:
```
p = parameters(period).person.age
```